cmd/scollector/collectors/iterable.go

```diff
@@ -0,0 +1,95 @@
+package collectors
+
+import (
+	"context"
+	"time"
+
+	"bosun.org/cmd/scollector/conf"
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+	"bosun.org/slog"
+	"github.com/bosun-monitor/statusio"
+)
+
+// iterable.io collector
+// to use add this in your scollector.toml:
+// [Iterable]
+//   StatusBaseAddr = "https://iterable.statuspage.io"
+//   # TsdbPrefix = "iterable.status."
+//   # MaxDuration = 3 # seconds
+
+type IterComp map[string]string
+
+func init() {
+	const (
+		defaultStatusBaseAddr = "https://iterable.statuspage.io"
+		defaultTsdbPrefix     = "iterable.status."
+		defaultMaxDuration    = 3 // Seconds
```

Comment:
If we do continue with the timeout, 3 second sseems like a short default

Reply:
What would you suggest? Should we trust data that is coming from a webpage that has taken more than several seconds to render? Probably not.